tests/Integration/Endpoints/AnalyticsPostsProxyEndpointTest.php

```diff
@@ -136,22 +136,26 @@ function () use ( &$dispatched ) {
 			(object) array(
 				'data' => array(
 					(object) array(
-						'author'  => 'Aakash Shah',
-						'date'    => wp_date( $date_format, strtotime( '2020-04-06T13:30:58' ) ),
-						'id'      => 'https://blog.parse.ly/web-analytics-software-tools/?itm_source=parsely-api',
-						'dashUrl' => PARSELY::DASHBOARD_BASE_URL . '/blog.parsely.com/find?url=https%3A%2F%2Fblog.parse.ly%2Fweb-analytics-software-tools%2F%3Fitm_source%3Dparsely-api',
-						'title'   => '9 Types of Web Analytics Tools — And How to Know Which Ones You Really Need',
-						'url'     => 'https://blog.parse.ly/web-analytics-software-tools/?itm_source=parsely-api',
-						'views'   => 142,
+						'author'         => 'Aakash Shah',
+						'date'           => wp_date( $date_format, strtotime( '2020-04-06T13:30:58' ) ),
+						'id'             => 'https://blog.parse.ly/web-analytics-software-tools/?itm_source=parsely-api',
+						'dashUrl'        => PARSELY::DASHBOARD_BASE_URL . '/blog.parsely.com/find?url=https%3A%2F%2Fblog.parse.ly%2Fweb-analytics-software-tools%2F%3Fitm_source%3Dparsely-api',
+						'thumbUrlMedium' => 'https://images.parsely.com/XCmTXuOf8yVbUYTxj2abQ4RSDkM=/85x85/smart/https%3A//blog.parse.ly/wp-content/uploads/2021/06/Web-Analytics-Tool.png%3Fw%3D150%26h%3D150%26crop%3D1',
+						'title'          => '9 Types of Web Analytics Tools — And How to Know Which Ones You Really Need',
+						'url'            => 'https://blog.parse.ly/web-analytics-software-tools/?itm_source=parsely-api',
+						'views'          => 142,
+						'postId'         => 0,
```

Comment:
Those are remote URLs we are testing, so these posts do not exist in the local WP instance. As such, `url_to_postid` will always return 0 here. We could probably create some local WP posts and test with that. This would need rework though, so we could do that as part of the tests PR.